packages/auth/lib/index.d.ts

```diff
@@ -342,12 +342,12 @@ export namespace FirebaseAuthTypes {
      * Returns the timestamp at which this account was created as dictated by the server clock
      * as an ISO Date string.
      */
-    creationTime?: string;
+    creationTime?: number;
```

Comment:
Is this returning a number for both Android & iOS? The Firebase Web SDK returns a string (ISO Date string) so this is what our implementation should be returning rather than a number